examples/embed-directive/embed-directive.go

```diff
@@ -0,0 +1,51 @@
+// Starting in Go version 1.16+, Go supports use of `//go:embed`
+// which is a directive to embed files and folders into
+// the application binary by the compiler.
+
+// Since Go does not have preprocessor or macros like C type of languages,
+// `pragmas/directives` are implemented by the compiler as comments `//`.
+// Using `//go:embed` tells the compiler to include arbitrary files/folders in binary.
+package main
+
+// If you are not using any exported identifiers from `embed` package directly,
+// you can add an underscore `_` before the package name, ask Go to import `embed`
+// without directly using it, like this: `_ "embed"`. In this example, we remove the
```

Comment:
We don't "remove the underscore"... we just import it because we use the name

Reply:
Valid callouts.

Reply:
Changed sentence.